src/build/emitter.ts

```diff
@@ -509,6 +509,11 @@ export function emitWebIdl(
       return objDomType;
     }
 
+    // If it's a quoted string literal (has " " inside), just return it
+    if (typeof objDomType === "string" && objDomType.includes('"')) {
```

Comment:
This implies that something went wrong... This shouldn't be needed.

Reply:
Perhaps we should use overrideType for that?

Reply:
This is still valid

Reply:
Why add this when we have already existing things that work 🤔

`type` is for IDL types, non-IDL types can go to `overrideType`.